node/config/types.go

```diff
@@ -484,6 +484,7 @@ type Pubsub struct {
 type Chainstore struct {
 	EnableSplitstore bool
 	Splitstore       Splitstore
+	BadgerVersion    int
```

Comment:
```suggestion
	// EXPERIMENTAL FEATURE. USE WITH CAUTION
	// BadgerVersion switches the version of the Badger database engine. The default is 2 which has
	// been well-tested and widely deployed. Switching this to version 4 will enable the new version
	// of Badger, but the blockstore will be incompatible with Lotus running with version 2. Switching
	// versions can only be done when the blockstore is empty and will be repopulated from a snapshot
	// or chain sync. It cannot be upgraded or downgraded with existing data and there is currently
	// no automatic migration tooling.
	// This is an experimental feature and should not be used in production.
	BadgerVersion    int
	EnableSplitstore bool
	Splitstore       Splitstore
```

moving it up is going to help readability since `Splitstore` is a large nested block

Reply:
thank you! I appreciate the addition of this comment